src/pages/Home/sections/Banner/Banner.module.scss

```diff
@@ -0,0 +1,195 @@
+
+$bg-shade1: #0f2027;
+$bg-shade2: #2c5364;
+$bg-shade3: #203a43;
+$btn-hover-bg: #0d739eab;
+
+
+.Home {
+  background: -webkit-linear-gradient(to right, $bg-shade1 $bg-shade2,$bg-shade3);
+  background: linear-gradient(to right, $bg-shade1,$bg-shade2,$bg-shade3);
+  color: colors.$white;
+  height: 89vh;
+  padding: 1rem;
+  padding-top: 3rem;
+
+  .content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    padding: 1rem 5rem;
+
+    .text {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      width: 60%;
+      text-align: center;
+
+      div {
+        font-family: "Dancing Script", cursive;
+        font-size: 4.85rem;
+      }
+      .button {
+        display: inline-block;
+        padding: 0.75rem 1.25rem;
+        border-radius: 2rem;
+        color: colors.$white !important;
+        text-transform: uppercase;
+        font-size: 1rem;
+        font-weight: semibold;
+        letter-spacing: 0.15rem;
+        transition: all 0.3s;
+        position: relative;
+        overflow: hidden;
+        z-index: 1;
+        border: none;
+        cursor: pointer;
+        margin-top: 15px;
+        z-index: 1001;
+        outline: none;
+
+        &:after {
+          content: "";
+          position: absolute;
+          bottom: 0;
+          left: 0;
+          width: 100%;
+          height: 100%;
+          background-color: $bg-shade3;
+          z-index: -2;
+        }
+        &:before {
+          content: "";
+          position: absolute;
+          bottom: 0;
+          left: 0;
+          width: 0%;
+          height: 100%;
+          background-color: $btn-hover-bg;
+          transition: all 0.3s;
+          z-index: -1;
+        }
+        &:hover {
+          color: colors.$white;
+          text-decoration: none;
+
+          &:before {
+            width: 100%;
+          }
+        }
+      }
+    }
+    .image {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      width: 40%;
+      margin-bottom: 1rem;
+
+      img {
+        width: 300px;
+      }
+    }
+  }
+  .wave {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 90px;
+    background: url("./../../../../assets/images/home-banner/wave.png");
+    background-size: 1000px 100px;
+  }
+  .wave1 {
+    animation: animate 30s linear infinite;
+    z-index: 1000;
+    opacity: 1;
+    animation-delay: 0s;
+    bottom: 0;
+  }
+  .wave2 {
+    animation: animate2 15s linear infinite;
+    z-index: 999;
+    opacity: 0.5;
+    animation-delay: -5s;
+    bottom: 10px;
+  }
+  .wave3 {
+    animation: animate 30s linear infinite;
+    z-index: 998;
+    opacity: 0.2;
+    animation-delay: -2s;
+    bottom: 15;
+  }
+  .wave4 {
+    animation: animate2 5s linear infinite;
+    z-index: 997;
+    opacity: 0.7;
+    animation-delay: -5s;
+    bottom: 20px;
+  }
+}
+
+@keyframes animate {
```

Comment:
why two similar animations have same name

Reply:
`animate` and `animate2 are different. see carefully. `background-position-x` is different